docker/setup_demo.sh

```diff
@@ -0,0 +1,15 @@
+# Create host mount directory and copy
+mkdir -p /tmp/hadoop_name
```

Comment:
add a protective `rm -rf` here? otherwise NN will start in safe mode since it does not recognize the data

Reply:
once the docker containers are stopped, the  files are deleted using "rm -rf ".  I did not want to do the deletion before docker compose down so that docker containers can be shutdown more cleanly. Will chat f2f to explain more.